src/observable/throw.ts

```diff
@@ -4,7 +4,7 @@ import {Subscription} from '../Subscription';
 
 export class ErrorObservable<T> extends Observable<T> {
 
-  static create<T>(error: T, scheduler?: Scheduler) {
+  static create<T>(error: any, scheduler?: Scheduler) {
```

Comment:
Got point. It can be bit confused by type signature looks as

`Observable<T>.throw<{}>()`

class type information is not being used as you've mentioned, but usecase wise it needs to remain as-is.